packages/layout/src/Legend.ts

```diff
@@ -222,20 +222,51 @@ export class Legend extends SVGWidget {
         const ordinal = d3ScaleOrdinal()
             .domain(dataArr.map(row => row[1]))
             .range(dataArr.map(row => row[0]));
-
+        let total = 0;
         this._legendOrdinal
             .orient(this.orientation())
             .title(this.title())
             .scale(ordinal)
+            .labels(d => {
+                if (this.showSeriesTotal()) {
+                    const val = this.data()[d.i].slice(1).reduce((acc, n) => acc + n, 0);
+                    total += val;
+                    return `${d.domain[d.i]} (${val})`;
+                }
+                return d.domain[d.i];
+            })
             ;
         this._g.call(this._legendOrdinal);
 
         this.updateDisabled(element, dataArr);
 
+        this.removeTotal();
         const bbox = this.getBBox(true, true);
         this._g.attr("transform", `translate(${this.width() / 2 - bbox.width / 2 + 5},${this.height() / 2 - bbox.height / 2})`);
+        if (this.showLegendTotal()) {
+            this.insertTotal(total, bbox);
+        }
```

Comment:
bleh 2

Reply:
How about:
```
this._g.selectAll(".legend-total").data(this.showLegendTotal() ? [total] : [])
```

Reply:
I'll try adding this, but the remove/insert paradigm allows us to use bbox for the placement. I'll try inserting the total as if it was a typical legend "cell" with your suggested selectAll/data pattern.